apps/backoffice-v2/src/domains/workflow-definitions/fetchers.ts

```diff
@@ -0,0 +1,71 @@
+import { ProcessStatuses, WorkflowDefinitionVariant } from '@ballerine/common';
+import { ObjectWithIdSchema } from '@/lib/zod/utils/object-with-id/object-with-id';
+import { z } from 'zod';
+import { apiClient } from '@/common/api-client/api-client';
+import { Method } from '@/common/enums';
+import { handleZodError } from '@/common/utils/handle-zod-error/handle-zod-error';
+import { env } from '@/common/env/env';
+import { getOriginUrl } from '@/common/utils/get-origin-url/get-url-origin';
+
+export const WorkflowDefinitionConfigSchema = z
+  .object({
+    isManualCreation: z.boolean().default(false),
+  })
+  .passthrough()
+  .nullable();
+
+export const PluginSchema = z.object({
+  name: z.string(),
+  displayName: z.string().or(z.undefined()),
+  status: z.enum(ProcessStatuses),
+});
+
+export type TPlugin = z.infer<typeof PluginSchema>;
+
+export const WorkflowDefinitionByIdSchema = ObjectWithIdSchema.extend({
+  name: z.string(),
+  version: z.number(),
+  variant: z.string().default(WorkflowDefinitionVariant.DEFAULT),
+  contextSchema: z.record(z.any(), z.any()).nullable(),
+  documentsSchema: z.array(z.any()).optional().nullable(),
+  config: WorkflowDefinitionConfigSchema,
+  definition: z
+    .object({
+      // transitionSchema: z
+      //   .array(
+      //     z.object({
+      //       state: z.string(),
+      //       schema: z.record(z.any(), z.unknown()),
+      //       additionalParameters: z.record(z.any(), z.unknown()).optional(),
+      //     }),
+      //   )
+      //   .nullable(),
```

Comment:
Copied from https://github.com/ballerine-io/ballerine/pull/1949 - commented until that PR is in.